madmom/features/beats.py

```diff
@@ -390,6 +390,51 @@ def recursive(position):
     return np.array(positions)
 
 
+def detect_beats_with_ccf(activations, interval, look_aside=0.2):
```

Comment:
I am actually not sure if you want to keep this method :) It is completely interchangeable with the detect_beats() method but seems to produce results which are a little bit worse. On the other hand its much faster. Maybe its an option for the BeatDetector (with constant tempo).